.travis.yml

```diff
@@ -60,9 +60,21 @@ matrix:
     - env: SYMFONY_VERSION=3.0.*
     - env: SYMFONY_VERSION=dev-master
 
+before_install: phpenv config-rm xdebug.ini
 install:
   - composer self-update
-  - composer require symfony/framework-bundle:${SYMFONY_VERSION} --prefer-dist
+  - composer require --prefer-dist \
+      symfony/framework-bundle:${SYMFONY_VERSION} \
+      symfony/console:${SYMFONY_VERSION} \
+      symfony/config:${SYMFONY_VERSION} \
+      symfony/dependency-injection:${SYMFONY_VERSION} \
+      symfony/event-dispatcher:${SYMFONY_VERSION} \
+      symfony/http-kernel:${SYMFONY_VERSION} \
+      symfony/process:${SYMFONY_VERSION} \
+      symfony/templating:${SYMFONY_VERSION} \
+      symfony/filesystem:${SYMFONY_VERSION} \
+      symfony/http-foundation:${SYMFONY_VERSION} \
+      symfony/routing:${SYMFONY_VERSION}
```

Comment:
For this I would add a job that does not define the `$SYMFONY_VERSION` env var and in that case simply run `composer install`.